### PR TITLE
client/Windows/wf_cliprdr.c: handle malloc error

### PR DESCRIPTION
**Pull request by chipitsine**

### CODE REVIEW

File: client/Windows/wf_cliprdr.c [hunk 1]
```diff
@@ -2133,6 +2133,8 @@ wf_cliprdr_server_format_data_request(CliprdrClientContext* context,
 			globlemem = (char*)GlobalLock(hClipdata);
 			size = (int)GlobalSize(hClipdata);
 			buff = malloc(size);
+			if (buff == NULL)
```
Comment (akallabeth) [inline client/Windows/wf_cliprdr.c:2136]: missing error handling

**Review by akallabeth — CHANGES_REQUESTED** [review]
* missing error handling (e.g. `CloseClipboard` 
* `globlemem`  is also not checked for `NULL`
* I believe the root issue you wanted to fix is a `size == 0` case? that is also not handled properly

**Comment by chipitsine** [comment]
`I believe the root issue you wanted to fix is a size == 0 case? that is also not handled properly` - no, I caught that error from cppcheck. I'll have a look at suggestions as well

**Comment by akallabeth** [comment]
@freerdp-bot test please

**Review by akallabeth — APPROVED** [review]

**Pull request merged**